### PR TITLE
flu page draft

### LINKED ISSUE #60: Design and implement "Flu and Other" page
page should contain non-covid related resources. At this time this page will have a banner to indicate it's WIP till v2 where we'll be redesigning the nav to contain all DELPHI resources.

### PR DESCRIPTION
**Pull request by sgratzl**
Closes #60 

image

based on the old index page

### CODE REVIEW

File: content/flu/_index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+We have participated, and [have done very well](http://www.cs.cmu.edu/~roni/CDC%20Flu%20Challenge%202014-2018%20Results.pdf), in all epidemiological forecasting challenges organized by the US government to date:
+
+<!--            : 2013--2014, 2014--2015 (winner),
+    <a target="_blank" rel="noopener" href="https://www.cdc.gov/flu/spotlights/flu-activity-forecasts-2016-2017.htm">2015--2016 (winner)</a>
+    ,
+    <a target="_blank" rel="noopener" href="https://predict.phiresearchlab.org/post/57f3f440123b0f563ece2576">2016--2017 (winner)</a>
+    <a target="_blank" rel="noopener" href="https://www.cdc.gov/flu/weekly/flusight/index.html#nav-group-aeff9">2017--2018 (winner)</a>-->
+### [Forecasting Seasonal Influenza in the US](https://www.cdc.gov/flu/weekly/flusight/)
+by [CDC](https://www.cdc.gov)): 2013 -- current
```
Comment (tildechris) [inline content/flu/_index.md:21]: This renders as a frowny face:

"by CDC): 2013 – current"

**Comment by tildechris** [comment]
The only thing that sticks out to me here is the link styles.  @sgratzl can you mirror the black/underlined style from the spec?

My initial thought was should we define a site wide link style, since the blogs use a different style, but I think it's OK to have a slightly different style here.

**Comment by sgratzl** [comment]
> The only thing that sticks out to me here is the link styles. @sgratzl can you mirror the black/underlined style from the spec?

the style is the default one when rendering Markdown. I can adapt them but we should consider setting some global styles which then will also be applied to the blog links

**Comment by sgratzl** [comment]
@tildechris can we also merge that one, please and fine tune later

**Comment by tildechris** [comment]
> @tildechris can we also merge that one, please and fine tune later

Merge this one, #61?  I agree this is close enough to merge.

**Review by tildechris — APPROVED** [review]
LGTM.

**Pull request merged**